dozer/cogs/moderation.py

```diff
@@ -627,6 +710,21 @@ class GuildMessageLinks(db.DatabaseObject):
     guild_id = db.Column(db.Integer, primary_key=True)
     role_id = db.Column(db.Integer, nullable=True)
 
+class PunishmentTimerRecord(db.DatabaseObject):
+    __tablename__ = "punishment_timers"
+    id = db.Column(db.Integer, primary_key=True)
+    guild_id = db.Column(db.Integer)
+    actor_id = db.Column(db.Integer)
+    target_id = db.Column(db.Integer)
+    orig_channel_id = db.Column(db.Integer, nullable=True)
+    type = db.Column(db.Integer)
+    reason = db.Column(db.String, nullable=True)
+    target_ts = db.Column(db.Integer)
+
+    type_map = {
+        1: Mute,
```

Comment:
Since you have these as the `type` attribute of the class, you can use that attribute instead of reusing the magic number here. That creates some repetition of the class names, which can be solved by using a dictionary comprehension for this.